src/fixes/ending_blank_line.rs

```diff
@@ -0,0 +1,114 @@
+use super::Fix;
+use crate::common::*;
+
+pub(crate) struct EndingBlankLineFixer<'a> {
+    name: &'a str,
+}
+
+impl Default for EndingBlankLineFixer<'_> {
+    fn default() -> Self {
+        Self {
+            name: "EndingBlankLine",
+        }
+    }
+}
+
+impl Fix for EndingBlankLineFixer<'_> {
+    fn name(&self) -> &str {
+        self.name
+    }
+
+    fn fix_warnings(
+        &self,
+        warnings: Vec<&mut Warning>,
+        lines: &mut Vec<LineEntry>,
+    ) -> Option<usize> {
+        let file = lines.first()?.file.clone();
+        let last_line = lines.last()?;
+
+        if !last_line.raw_string.ends_with(LF) {
+            lines.push(LineEntry {
+                number: lines.len() + 1,
+                file,
+                raw_string: LF.to_string(),
+            });
+        }
+
+        for warning in warnings {
+            if warning.check_name == self.name {
```

Comment:
You don't neet to check the `check_name` here, because it is already checked there:
https://github.com/dotenv-linter/dotenv-linter/blob/0cebfe3a6f4cbaf76b939ed9689c47cbc6f3ddaf/src/fixes.rs#L61

Reply:
done!